pkg/namesgenerator/names-generator.go

```diff
@@ -248,19 +248,19 @@ var (
 
 		// Asima Chatterjee was an Indian organic chemist noted for her research on vinca alkaloids, development of drugs for treatment of epilepsy and malaria - https://en.wikipedia.org/wiki/Asima_Chatterjee
 		"chatterjee",
-
+		
```

Comment:
I think `gofmt` doesn't like the extra empty line;

```
pkg/namesgenerator/names-generator.go:125: File is not `goimports`-ed (goimports)
```

Let me run `gofmt` and push